lib/cli/options.js

```diff
@@ -192,8 +196,7 @@ const loadPkgRc = (args = {}) => {
           filepath
         );
       } else {
-        debug('failed to read default package.json at %s; ignoring',
-              filepath);
+        debug('failed to read default package.json at %s; ignoring', filepath);
```

Comment:
[Style] Please keep these and other unrelated style changes out of the PR. It makes it harder to know what's actually different.